app/src/main/java/de/westnordost/streetcomplete/quests/postbox_collection_times/AddCollectionTimesForm.kt

```diff
@@ -52,7 +54,29 @@ class AddCollectionTimesForm : AbstractQuestFormAnswerFragment<CollectionTimesAn
         collectionTimesList.isNestedScrollingEnabled = false
         checkIsFormComplete()
 
-        addTimesButton.setOnClickListener { collectionTimesAdapter.addNew() }
+        addTimesButton.setOnClickListener { onClickAddButton(it) }
+    }
+
+    private fun onClickAddButton(v: View) {
+        val rows = collectionTimesAdapter.collectionTimesRows
+
+        val addTimeAvailable = rows.isNotEmpty() && rows.last() is WeekdaysTimesRow
```

Comment:
all rows of the `CollectinTimesAdapter` are `WeekdaysTimesRow`s, no need to check for that

Reply:
Resolved with e2a6116